handlers.go

```diff
@@ -152,9 +152,13 @@ var CreateLogFilesHandler = Handler{
 		)
 
 		if m.Cfg.FifoLogWriter != nil {
-			if err := captureFifoToFile(m.logger, logFifoPath, m.Cfg.FifoLogWriter); err != nil {
-				m.logger.Warnf("captureFifoToFile() returned %s. Continuing anyway.", err)
-			}
+			// We wrap this in a goroutine since opening the fifo queue will block
+			// until the fifo has been connected when starting firecracker
+			go func() {
+				if err := captureFifoToFile(m.logger, logFifoPath, m.Cfg.FifoLogWriter); err != nil {
```

Comment:
There is a goroutine already in `captureFifoToFile`. It may be best to just remove the one in `captureFifoToFile` and make `captureFifoToFile` to be expected to be ran in a goroutine.